nyx/nyx_gui/frontend/gui_tools_partition_manager.c

```diff
@@ -1720,12 +1748,74 @@ static lv_res_t _create_mbox_partitioning_next(lv_obj_t *btn)
 
 	lv_obj_align(mbox, NULL, LV_ALIGN_CENTER, 0, 0);
 	lv_obj_set_top(mbox, true);
+	manual_system_maintenance(true);
 
 	free(txt_buf);
 
 	return LV_RES_OK;
 }
 
+static lv_res_t _create_mbox_partitioning_android(lv_obj_t *btns, const char *txt)
+{
+	int btn_idx = lv_btnm_get_pressed(btns);
+
+	mbox_action(btns, txt);
+
+	switch (btn_idx)
+	{
+		case 0:
+			part_info.and_dynamic = true;
+			_create_mbox_partitioning_warn(NULL);
+			break;
+		case 1:
+			part_info.and_dynamic = false;
+			_create_mbox_partitioning_warn(NULL);
+			break;
+		default:
+			break;
+	}
+
+	return LV_RES_OK;
+}
+
+static lv_res_t _create_mbox_partitioning_andr_part(lv_obj_t *btn)
+{
+	lv_obj_t *dark_bg = lv_obj_create(lv_scr_act(), NULL);
+	lv_obj_set_style(dark_bg, &mbox_darken);
+	lv_obj_set_size(dark_bg, LV_HOR_RES, LV_VER_RES);
+
+	static const char *mbox_btn_map[] = { "\222Dynamic", "\222Legacy", "\222Cancel", "" };
+	lv_obj_t * mbox = lv_mbox_create(dark_bg, NULL);
+	lv_mbox_set_recolor_text(mbox, true);
+
+	lv_obj_set_width(mbox, LV_HOR_RES / 9 * 6);
+	lv_mbox_set_text(mbox, "#FF8000 Partition Manager#");
+
+	lv_obj_t *lbl_status = lv_label_create(mbox, NULL);
+	lv_label_set_recolor(lbl_status, true);
+
+	lv_label_set_text(lbl_status,
+		"Please select an Android partition scheme:\n\n"
+		"#FFDD00 Android 13 and later:# Dynamic partitions\n"
+		"#FFDD00 Android 10-11:# Legacy (system-as-root) partitions\n");
```

Comment:
You are describing the buttons, so
`#C7EA46 Dynamic:# Android 13 or later\n`
`#C7EA46 Legacy:# Android 10-11`

Or the whole thing can just be a `#C7EA46 Please select Android partition scheme.#`
And have the buttons as `Android 13+`, `Android 10-11`
Since no one reads the descriptions and dynamic/legacy means nothing to the user.

Color should be `C7EA46`